From b7679594f3b8cb5cb8f7b18e4dac5aef35b5aadd Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Sun, 9 Dec 2018 13:08:56 -0700
Subject: [PATCH 42/48] Revert "PhoneWindowManager: Check if proposed rotation
 is in range"

* This was accidentally merged

This reverts commit 4262768f3102a3df8670383f57cd24c11e46d005.

Change-Id: I4ab51c626040ea0e89d99ac61c6570f46e4a38ce
---
 .../core/java/com/android/server/policy/PhoneWindowManager.java | 2 +-
 .../com/android/server/policy/WindowOrientationListener.java    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/services/core/java/com/android/server/policy/PhoneWindowManager.java b/services/core/java/com/android/server/policy/PhoneWindowManager.java
index 71d49a903fa..f4e2b788d9c 100644
--- a/services/core/java/com/android/server/policy/PhoneWindowManager.java
+++ b/services/core/java/com/android/server/policy/PhoneWindowManager.java
@@ -8003,7 +8003,7 @@ public class PhoneWindowManager implements WindowManagerPolicy {
 
         synchronized (mLock) {
             int sensorRotation = mOrientationListener.getProposedRotation(); // may be -1
-            if (sensorRotation < 0 || sensorRotation > 3) {
+            if (sensorRotation < 0) {
                 sensorRotation = lastRotation;
             }
 
diff --git a/services/core/java/com/android/server/policy/WindowOrientationListener.java b/services/core/java/com/android/server/policy/WindowOrientationListener.java
index 0e151d697c4..9b4251def93 100644
--- a/services/core/java/com/android/server/policy/WindowOrientationListener.java
+++ b/services/core/java/com/android/server/policy/WindowOrientationListener.java
@@ -1084,7 +1084,7 @@ public abstract class WindowOrientationListener {
 
         public int evaluateRotationChangeLocked() {
             unscheduleRotationEvaluationLocked();
-            if (mDesiredRotation == mProposedRotation || mDesiredRotation > 3) {
+            if (mDesiredRotation == mProposedRotation) {
                 return -1;
             }
             final long now = SystemClock.elapsedRealtimeNanos();
-- 
2.17.1

